applications/osb-portal/src/service/UserService.tsx

```diff
@@ -91,3 +92,7 @@ export async function register() {
 const errorCallback = (error: any) => {
     initApis(null);
 }
+
+export function canEditWorkspace(user: UserInfo, workspace: Workspace) {
```

Comment:
this should be in line with the backend permission/rules for editing workspaces
imo it's better to centralize the business logic of is the user allowed to edit on the backend
it could be done by returning an extra attribute in the workspace model: is_editable true/false

Reply:
Hadn't really thought about moving that logic to the backend, agree that would be preferrable bot not really a game changer so I would do that only if we are touching this logic also for other reasons

Reply:
agree